content/building/environment-variable-groups.md

```diff
@@ -16,11 +17,13 @@ workflows:
         - other_values
 ```
 
-Variables defined in environment variable groups work exactly as [Environment Variables](../building/environment-variables/#using-environment-variables). The value of a variable named `API_TOKEN` can be referenced in a workflow as `$API_TOKEN`. Variables defined with the ***secure*** option will have values obfuscated in the Codemagic UI.
+Variables defined in environment variable groups work exactly as [Environment Variables](../building/environment-variables/#using-environment-variables). The value of a variable named `API_TOKEN` can be referenced in a workflow as `$API_TOKEN`. Variables defined with the **_secure_** option will have values obfuscated in the Codemagic UI.
 
 ## Global variables and secrets
 
-Global variable groups can be defined on the team settings page (which you can navigate to for your team on the [Teams page](https://codemagic.io/teams)). Variable groups defined here can be used in any codemagic.yaml workflow in any application of the team.
+Global variable groups can be defined on the team settings page (which you can navigate to for your team on the [Teams page](https://codemagic.io/teams)).
+
+By default, variable groups defined here can be used in any codemagic.yaml workflow in any application of the team. It is possible to limit the variable groups access to specific applications by clicking the edit icon next to the group you wish to manage under **Application access**.
```

Comment:
for clarity: It is possible to limit access to variable groups to specific applications by..